application/utils/harvester/git_repository_client.py

```
@@ -0,0 +1,184 @@
+            raise
+
+    def checkout(self, reference: str) -> None:
+        logger.info(
+            "Checking out %s in %s/%s",
+            reference,
+            self.owner,
+            self.repository,
+        )
+
+        try:
+            subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)